src/server/game/Entities/Player/Player.cpp

```diff
@@ -1253,7 +1253,7 @@ void Player::ToggleAFK()
     ToggleFlag(PLAYER_FLAGS, PLAYER_FLAGS_AFK);
 
     // afk player not allowed in battleground
-    if (isAFK() && InBattleground() && !InArena())
+    if (isAFK() && InBattleground() && InArena())
```

Comment:
This breaks `/afk` in battlegrounds.

Reply:
> This breaks `/afk` in battlegrounds.

Hmmm i didnt test this pr, i will check it out.